ios/RCTMGL/RCTMGLMapViewManager.m

```diff
@@ -95,6 +95,28 @@ - (UIView *)view
 
 #pragma mark - React Methods
 
+RCT_EXPORT_METHOD(getPointInView:(nonnull NSNumber*)reactTag
+                  atCoordinate:(NSArray<NSNumber*>*)coordinate
+                  resolver:(RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject)
+{
+    [self.bridge.uiManager addUIBlock:^(__unused RCTUIManager *manager, NSDictionary<NSNumber*, UIView*> *viewRegistry) {
+        id view = viewRegistry[reactTag];
+        
+        if (![view isKindOfClass:[RCTMGLMapView class]]) {
+            RCTLogError(@"Invalid react tag, could not find RCTMGLMapView");
+            return;
+        }
+        
+        RCTMGLMapView *reactMapView = (RCTMGLMapView*)view;
+
+        CGPoint pointInView = [reactMapView convertCoordinate:CLLocationCoordinate2DMake([coordinate[0] doubleValue], [coordinate[1] doubleValue])
```

Comment:
Would you mind updating coordinates passed in to be in GeoJSON format [longitude, latitude]?

Reply:
Thanks for reviewing. I changed the order in the last commit.

Reply:
Great, I'll test it out today and then we can merge it in